doc/api/stream.md

```diff
@@ -271,68 +487,64 @@ readable.resume()
 readable.isPaused() // === false
 ```
 
-#### readable.pause()
+##### readable.pause()
 
 * Return: `this`
 
-This method will cause a stream in flowing mode to stop emitting
-[`'data'`][] events, switching out of flowing mode. Any data that becomes
-available will remain in the internal buffer.
+The `readable.pause()` method will cause a stream in flowing mode to stop
+emitting [`'data'`][] events, switching out of flowing mode. Any data that
+becomes available will remain in the internal buffer.
```

Comment:
I'm not sure where the best place to put this admonition is, but we should really note that consumers should only use one of the two public APIs at a time — either use `.pause()`, `.resume()`, `.read()`, `.on('readable')`, `.on('data')`, _or_ use `.pipe()`. Generally folks should prefer `.pipe()`, but if they need more fine-grained control over the transfer and generation of data, they should use the former API. In no cases should use be mixed, however, since they both manipulate the same state machine.